app/controllers/application_controller.rb

```diff
@@ -10,6 +10,8 @@ class ApplicationController < ActionController::Base
   include ThreeScale::OnPremises
   include ProxyConfigAffectingChanges::ControllerExtension
 
+  add_flash_types :default, :info, :success, :warning, :danger
```

Comment:
@josemigallas I believe, this is the line that makes the `/default.xml` endpoints fail. Probably because this method adds a `default` method on the controller, and overrides the one defined manually?

I guess ideally we'd define this in `FrontendController`, but I can see that this controller relies on flashes itself? (probably this method should also be extracted to `FrontendController`?

Reply:
Yes, but I have another suggestion here: https://github.com/3scale/porta/pull/4085/files :grimacing:

Reply:
And it's greeeeeeen
![image](https://github.com/user-attachments/assets/f625133c-aaf3-43e3-88e3-2a5a33ddf57d)